src/cephadm/cephadm.py

```diff
@@ -46,7 +46,7 @@
 # Default container images -----------------------------------------------------
 DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:main'
 DEFAULT_IMAGE_IS_MAIN = True
-DEFAULT_IMAGE_RELEASE = 'reef'
+DEFAULT_IMAGE_RELEASE = 'squid'
```

Comment:
just a note, the location of this constant is being moved as part of https://github.com/ceph/ceph/pull/53052. If that works get merged before this it might cause a conflict so wanted you to be aware of it. We plan to add a commit to that PR to change the release checklist to point to the new location once it's finalized as well.

Reply:
Thanks, Adam!

Reply:
this did end up getting merged. We've updated the checklist in the docs with the new location of this constant as part of that PR